---
tags: üì•Ô∏è/üìúÔ∏è/üü•Ô∏è
publish: true
aliases: 
  - Genome-scale pan-cancer interrogation of lncRNA dependencies using CasRx
  - monteroGenomescalePancancerInterrogation2024
---

> [!link]-
> zotero_link:: [ÂÖ®Êñá](zotero://select/library/items/P6DJHSNM)

> [!cite]-
> citekey:: monteroGenomescalePancancerInterrogation2024

> [!abstract]-
> abstract:: Abstract
            Although long noncoding RNAs (lncRNAs) dominate the transcriptome, their functions are largely unexplored. The extensive overlap of lncRNAs with coding and regulatory sequences restricts their systematic interrogation by DNA-directed perturbation. Here we developed genome-scale lncRNA transcriptome screening using Cas13d/CasRx. We show that RNA targeting overcomes limitations inherent to other screening methods, thereby considerably expanding the explorable space of the lncRNAome. By evolving the screening system toward pan-cancer applicability, it supports molecular and phenotypic data integration to contextualize screening hits or infer lncRNA function. We thereby addressed challenges posed by the enormous transcriptome size and tissue specificity through a size-reduced multiplexed gRNA library termed Albarossa, targeting 24,171 lncRNA genes. Its rational design incorporates target prioritization based on expression, evolutionary conservation and tissue specificity, thereby reconciling high discovery power and pan-cancer representation with scalable experimental throughput. Applied across entities, the screening platform identified numerous context-specific and common essential lncRNAs. Our work sets the stage for systematic exploration of lncRNA biology in health and disease.

> [!keywords]-
> keywords:: 

> [!authors]-
> authors:: Juan J. Montero, Riccardo Trozzo, Maya Sugden, Rupert √ñllinger, Alexander Belka, Ekaterina Zhigalova, Paul Waetzig, Thomas Engleitner, Marc Schmidt-Supprian, Dieter Saur, Roland Rad

> [!meta]-
> url:: https://www.nature.com/articles/s41592-024-02190-0
> doi:: 

> [!related]-


```dataview
TABLE created, updated as modified, tags, type
FROM ""
WHERE related != null
AND contains(related, "monteroGenomescalePancancerInterrogation2024")
```

> [!hypothesis]-
> hypothesis:: 

> [!methodology]- 
> methodology:: 

> [!result]- Result(s) 
> results::

> [!summary]- Summary of Key Points
> summary:: 

## Notes

| <mark class="hltr-grey">Highlight Color</mark> | Meaning                       |
| ---------------------------------------------- | ----------------------------- |
| <mark class="hltr-red">Red</mark>              | Disagree with Author          |
| <mark class="hltr-orange">Orange</mark>        | Important Point By Author     |
| <mark class="hltr-yellow">Yellow</mark>        | Interesting Point             |
| <mark class="hltr-green">Green</mark>          | Important To Me               |
| <mark class="hltr-blue">Blue</mark>            | Notes After Initial Iteration |
| <mark class="hltr-purple">Purple</mark>        | Literary Note To Lookup Later |

- <mark class="hltr-yellow">"The RNA from the lncRNA dependencies validation (Fig. 6g), and from the indiscriminate off-target RNA cleavage experiments (Fig. 1h,i) was processed as follows: library preparation for bulk RNA-seq was done as described previously96. Briefly, barcoded cDNA of each sample was generated with a Maxima reverse transcriptase polymerase (ThermoFisher) using an oligonucleotide-dT primer containing barcodes, unique molecular identifiers (UMIs) and an adapter; 5‚Ä≤-ends of the cDNAs were extended by a template switch oligonucleotide and full-length cDNA was amplified with primers binding to the template switch oligonucleotide-site and the adapter. An NEB UltraII FS kit was used to fragment cDNA. After end repair and A-tailing, a TruSeq adapter was ligated and 3‚Ä≤-end fragments were finally amplified using primers with Illumina P5 and P7 overhangs. In comparison with ref. 96, the P5 and P7 sites were exchanged to allow sequencing of the cDNA in read1 and barcodes and UMIs in read2 to achieve better cluster recognition. The library was sequenced on a NextSeq 500 (Illumina) system with 65 cycles for the cDNA in read1 and 16 cycles for the barcodes and UMIs in read2. Data were processed using the published Drop-seq pipeline (v.1.0) to generate sample- and genewise UMI tables97. Reference genome GRCh38 (ref. 93) was used for align- ment. Transcript and gene definitions were used according to the Gencode v.38 (ref. 10). Differential expression analysis was performed in R v.3.6.3 with DESeq2 v.1.26 (ref. 95), testing each treatment against the respective control condition.‚Äù</mark> [Page 15](zotero://open-pdf/library/items/P6DJHSNM?page=15&annotation=B26HRLJH) 
 
	- test111 
 
> [!context]-
> ==(How this article relates to other work in the field; how it ties in with key issues and findings by others, including yourself)==
> context:: 

> [!significance]-
> ==(to the field; in relation to your own work)==
> significance:: 
